packages/forma-36-react-components/src/components/Tabs/TabPanel.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+
+interface TabPanelProps
+  extends React.DetailedHTMLProps<
+    React.HTMLAttributes<HTMLDivElement>,
+    HTMLDivElement
+  > {
```

Comment:
Why do you need this? Just curious.

Reply:
Because API allows using (with `...rest`) all HTML properties that you would normally use with `<div />`